helm/tests/gateway/configmap_user_test.yaml

```diff
@@ -0,0 +1,53 @@
+suite: Test Gateway configmap section user
+templates:
+  - "gateway/gateway-configmap.yaml"
+tests:
+  - it: Test user config block
+    template: gateway/gateway-configmap.yaml
+    set:
+      userManagement:
+        email: test@test.com
+        password: password
+        name: name
+        username: luk
+        registration: 1
+        resetPassword: 2
+        blockedAccount: 3
+        mfaChallenge: 4
+        mfaVerifyAttempt: 5
+        webAuthn: 6
+    asserts:
+      - hasDocuments:
+          count: 1
+      - isKind:
+          of: ConfigMap
+      - matchRegex:
+          path: data.[gravitee.yml]
+          pattern: "user:\n( *[a-zA-Z]*: .*\n)* *email: test@test.com"
```

Comment:
**suggestion**: Looks like we can use a text block as regex. It should help for readability and indentation validation.
ex : https://github.com/gravitee-io/gravitee-api-management/blob/master/helm/tests/gateway/confiigmap_db_less_test.yaml#L31

```
      - matchRegex:
          path: data.[gravitee.yml]
          pattern: |
            email:
              policy:
                pattern: ...
```

Reply:
fixed